xwiki-commons-core/xwiki-commons-component/xwiki-commons-component-default/src/test/java/org/xwiki/component/JavaxProviderIntegrationTest.java

```diff
@@ -19,6 +19,9 @@
  */
 package org.xwiki.component;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
```

Comment:
According to https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HImports, static imports should be last, not first, so the change here seems wrong.

Reply:
Indeed, not sure how it ended there.